pandas/util/_print_versions.py

```diff
@@ -143,7 +142,7 @@ def show_versions(as_json: str | bool = False) -> None:
             sys.stdout.writelines(json.dumps(j, indent=2))
         else:
             assert isinstance(as_json, str)  # needed for mypy
-            with codecs.open(as_json, "wb", encoding="utf8") as f:
+            with open(as_json, "w", encoding="utf-8") as f:
```

Comment:
I changed it from "wb" to "w" because I think the "b" here is a bug (bytes *and* utf-8???)